cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/ReadThroughBookCache.java

```diff
@@ -78,7 +79,19 @@ public Collection<PageInfo> loadPageInfo(BookId bookId, boolean loadRemoved) {
         return result;
     }
 
-    public BookInfo loadBook (BookId bookId) throws CradleStorageException {
+    @Override
+    public Collection<PageInfo> loadPageInfo(BookId bookId, LocalDateTime start, boolean loadRemoved) {
```

Comment:
Maybe it should be an iterator instead of a collection? Why do we need to have all pages in memory?

Reply:
What do you think about change all collections to CradleResultSet?